README.md

```diff
@@ -183,10 +183,9 @@ identity provider, such as (2018/2) [solid.community](https://solid.community/),
 
 #### WebID_TLS
 
-A WebID profile from one of the Solid-compliant [identity providers](https://solid.github.io/solid-idps/), such as [databox.me](https://databox.me/),  
+WebID-TLS currently only works in Firefox at the moment (Oct 2018)
 
-With WebID-TLS, you will need to make a WebID browser certificate from the above profile (this is usually created
-when you sign up for a WebID profile account, but it only works on Firefox at the moment (2018)).
+To use WebID-TLS, you will need to make a WebID browser certificate from one of the Solid-compliant [identity providers](https://solid.github.io/solid-idps/) - this is usually created when you sign up for a WebID profile account.
```

Comment:
This probably has changed as well. @melvincarvalho What advice should we be giving people starting with client certs (presumably wrapped in WebID-OIDC)?

Reply:
@RubenVerborgh,

Are you referring to folks that already have an X.509 cert branded with a WebID? If so, they should simply test authentication using the following:

1.  http://id.myopenlink.net/ods/webid_demo.html -- Basic WebID-TLS protocol tester
1. https://kidehen3.openlinksw.com:8444  -- Solid Pod (supports WebID-TLS via popup.html option as part of Idp authentication flow)
1. https://osdb.openlinksw.com -- App that supports both OpenID and TLS 


/cc @melvincarvalho @timbl

Reply:
@kdiehen When I open http://id.myopenlink.net/ods/webid_demo.html with Chrome and click "Check", I do not get a cert prompt but get "Error - noCert'. Some things might have changed on my end after a system migration though.

Reply:
I have used the same cert for years and had an ever improving experience with WebID-TLS login, so my advice would be biased.  I would say once you are up and running you will have a great secure experience, for which I have used KEYGEN on databox, and scripts to generate.  

While open to the benefits of WebID-OIDC, but I have login issues with it on a daily basis, and a complex UX.  WebID-TLS wrapped in OIDC I find to be very difficult both on client and server.  Much better is direct WebID-TLS.

It all depends on your use case, I suppose.

Reply:
@kidehen The same.

@melvincarvalho Please create issues for any login problem you run into.

Reply:
@RubenVerborgh ,
What webid are you using? I assume the WebID situated in the SAN of your X.509 cert resolves to a WebID-Profile doc that includes a public key from the aforementioned X.509 cert? 

I don't have any problems at all. In short, I wouldn't be able to function without WebID-TLS (especially with the delegation feature) i.e., that's an integral part of how I work :)

Reply:
I just wonder why @perploug had the problem. Perhaps the client certificate was added to Firefox, but not to Chrome?

Reply:
@RubenVerborgh ,
On macOS the following browsers use keychain as their credentials store: Chrome, Safari, Opera. 
Across all platforms Firefox is still using its own credentials store which is something they were supposed to be moving away from.
Microsoft Edge uses the Windows keystore for storing credentials. 

Thus, on macOS credentials are either in keychain (easy to lookup since it is a core OS component) or in the Firefox credentials store. 

You can export (backup)  credentials from Firefox to a pkcs#12 document and then import the contents (i.e., credentials in question) of said document into  Keychain.

/cc @perploug